doc/release.md

```diff
@@ -149,8 +144,16 @@ git push --tags origin
 git push --delete origin v${new_version}${rc}
 ```
 
-3\. Upload wheels from GitHub Releases to PyPI (once [Travis is finished building them](https://travis-ci.com/github/dib-lab/sourmash/)).  You can manually download
-all the files from [the releases page], or, if you have [`hub`](https://hub.github.com/), you can use that to download the packages.
+3\. Upload wheels from GitHub Releases to PyPI
+
+[Travis will automatically build wheels and upload them to PyPI](https://travis-ci.com/github/dib-lab/sourmash/).
```

Comment:
Not PyPI, it will upload them to GitHub Releases. PyPI upload is still manual.

(It is not hard to change that and upload to PyPI, tho)

Reply:
fixed in 63ed029